pyproject.toml

```diff
@@ -10,6 +10,8 @@ dependencies = [
     "kfp-kubernetes>=1.4.0",
     "kfp==2.9.0",
     "kubernetes==30.1.0",
+    "model-registry==0.2.14",
```

Comment:
According to [Model Registry documentation](https://www.kubeflow.org/docs/components/model-registry/getting-started/#setup), you will need to include kserve too

Reply:
It's not required unless you are creating an inference endpoint from the registered model. My testing of registering a model doesn't require it.